packages/core/controls.mjs

```diff
@@ -819,13 +819,23 @@ const _setter = (func, name) =>
   };
 
 generic_params.forEach(([type, name, description]) => {
-  controls[name] = (...pats) => _name(name, ...pats);
+  controls[name] = function (...pats) {
+    const result = _name(name, ...pats);
+
+    // Add a function for composing this control with another pattern
+    result.__compose = function (pat) {
```

Comment:
shouldn't this be done inside _name, so createParam will get the same treatment?